run.sh

```diff
@@ -424,9 +418,57 @@ function add_scheduler(){
     echo -e "[INFO] [$(date +"%Y-%m-%d %H:%M:%S")] Added Job Scheduler for run Cloud Function."
 }
 
+ 
+# Init script with proper message and display active user account
+
+function gcloud_init_confs(){
+    user_active_account="$(gcloud auth list --filter=status:ACTIVE --format="value(account)")"
+    echo -e "[INFO] [$(date +"%Y-%m-%d %H:%M:%S")] Your active account [${user_active_account}]"
+    echo -e "[INFO] [$(date +"%Y-%m-%d %H:%M:%S")] Choose Project ID"
+    _choose_and_set_project_id
+	echo -e "[INFO] [$(date +"%Y-%m-%d %H:%M:%S")] Project ID was updated."
+}
+
+
+# Choose and set project id
+# Error:
+#   Exit Code 1
+function _choose_and_set_project_id(){
+    array_projects=()
+    project_id=""
+    count=0
+    for project in  $(gcloud projects list --format="value(projectId)")
+    do
+        count=$((count + 1))
+        echo "[$count]:  $project"
+        array_projects+=("$project")
+    done
+    read -n 2 -p "Please fill in number of the project where you would like the integration to be deployed in: " mainmenuinput
+    count_projects=0
+    for value in "${array_projects[@]}"
+    do
+        count_projects=$((count_projects + 1))
+        if [ "$mainmenuinput" = "$count_projects" ]; then
+            project_id=$value
+            gcloud config set project $project_id
+            if [[ $? -ne 0 ]]; then
+                echo -e "[ERROR] [$(date +"%Y-%m-%d %H:%M:%S")] Failed to create Cloud Function."
```

Comment:
Is there a way to print out why it failed to create?

Reply:
It's printing generic error in gcloud CLI, I just defining where is happens.